besu-plugins/linea-sequencer/sequencer/src/main/java/net/consensys/linea/rpc/services/LineaBundleEndpointsPlugin.java

```diff
@@ -76,6 +87,19 @@ public void doStart() {
     // set the pool
     lineaSendBundleMethod.init(bundlePoolService, createTransactionValidator());
     lineaCancelBundleMethod.init(bundlePoolService);
+
+    lineaTransactionPoolValidatorConfiguration =
+        Optional.of(transactionPoolValidatorConfiguration());
+    bundleDeniedAddresses.set(
+        lineaTransactionPoolValidatorConfiguration.get().bundleDeniedAddresses());
+  }
```

Comment:
As you can see from the previous code
```
new AllowedAddressValidator(
              transactionPoolValidatorConfiguration().bundleDeniedAddresses()),
```
here it is guaranteed that `transactionPoolValidatorConfiguration()` returns not null, so you can get rid of the optionality, that because if the configuration was not available then we would have already failed